nautobot/core/tests/test_authentication.py

```diff
@@ -24,7 +25,7 @@
 class ExternalAuthenticationTestCase(TestCase):
```

Comment:
Is it reasonable to call `get_user_model()` in each case, or would it work as well (and with fewer diffs) to do a global `User = get_user_model()` once at the start of each relevant file?

Reply:
Yes. That would be the correct way to do it since we're swapping the user model.

Reply:
I tend to agree with Glenn here. In the places like in the test code where `User` is being referenced a lot, I think it makes sense to put `User = get_user_model()` in the module namespace. I know it's not as straightforward as a search/replace but it will reduce repeated calls and keep the code a little more readable.